client/src/environments/environment.prod.ts

```diff
@@ -1,3 +1,4 @@
 export const environment = {
   production: true,
+  RECAPTACHA_SITE_KEY: '', // TODO For kfet-insa.fr domains
```

Comment:
It should be an env variable sent when building the front (I don't know how for now)